enzyme/tools/enzyme-tblgen/blas-tblgen.cpp

```diff
@@ -670,6 +670,8 @@ void emit_fwd_rewrite_rules(const TGPattern &pattern, raw_ostream &os) {
      << "    auto callval = call.getCalledValue();           \n"
      << "#endif                                            \n\n";
 
+  os << "    assert(!EnzymeRuntimeActivityCheck && \"RuntimeActivity for fwd-mode is not implemented yet!\");\n";
```

Comment:
Can this be changed to CustomErrorHandler and emitfailure like below?

https://github.com/EnzymeAD/Enzyme/blob/3c71081c998a0aef7454e4e3d31b1dfd58549bd6/enzyme/Enzyme/AdjointGenerator.h#L346

This way the compiler won't die, but instead say that it failed to compile